### PR TITLE
chore: slight updates to properly publish as a crate

### PR DESCRIPTION
**Pull request by ajewellamz**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: releases/rust/db_esdk/Cargo.toml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name = "aws-db-esdk"
 version = "0.1.0"
 edition = "2021"
 rust-version = "1.80.0"
-keywords = ["crypto", "cryptography", "security", "dynamodb", "ddb", "encryption", "client-side", "clientside"]
+keywords = ["cryptography", "security", "dynamodb", "encryption", "client-side"]
```
Comment (RitvikKapila) [inline releases/rust/db_esdk/Cargo.toml:6]: Do we want to make this and the dafny-runtime = "0.1.1: change to the `DynamoDbEncryption/runtimes/rust/Cargo.toml` as well?

File: releases/rust/db_esdk/Cargo.toml [hunk 2]
```diff
@@ -24,7 +24,7 @@ aws-sdk-kms = "1.50.0"
 aws-smithy-runtime-api = {version = "1.7.3", features = ["client"] }
 aws-smithy-types = "1.2.9"
 chrono = "0.4.38"
-dafny_runtime = { path = "./dafny_runtime_rust"}
+dafny-runtime = "0.1.0"
```
Comment (RitvikKapila) [inline releases/rust/db_esdk/Cargo.toml:27]: Is it v0.1.0 or v0.1.1?
```suggestion
dafny-runtime = "0.1.1"
```

**Review by RitvikKapila — COMMENTED** [review]
Left a couple of small comments. Otherwise LGTM

File: releases/rust/db_esdk/Cargo.toml (hunk 2 above)
Comment (ajewellamz) [reply releases/rust/db_esdk/Cargo.toml:27]: It doesn't matter, 0.1.1 was just for a typo in the README.
Specifying 0.1.0 will still get you 0.1.1 under most circumstances.

File: releases/rust/db_esdk/Cargo.toml (hunk 1 above)
Comment (ajewellamz) [reply releases/rust/db_esdk/Cargo.toml:6]: Oh, probably.

**Review by RitvikKapila — APPROVED** [review]
LGTM

**Pull request merged**